moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -161,7 +161,7 @@ class IKFastKinematicsPlugin : public kinematics::KinematicsBase
   const size_t num_joints_;
   std::vector<int> free_params_;
   bool active_;  // Internal variable that indicates whether solvers are configured and ready
-  const std::string name_{ "ikfast" };
+  const std::string name_{ "ikfast_kinematics_plugin" };
```

Comment:
I suggest to keep the short name `ikfast`. `moveit_kinematics` is part of the logger name anyway.